beard/metrics/text.py

```diff
@@ -162,3 +164,51 @@ def jaro_winkler(s1, s2, p=0.1):
             break
 
     return jaro_distance + p * common_prefix * (1 - jaro_distance)
+
+
+def levenshtein(a, b):
+    """Calculate the levenshtein distance between strings a and b.
+
+    Case sensitiveness is activated, meaning that uppercase letters
```

Comment:
Wouldn't it be useful to have case sensitiveness as a parameter in this function?